.github/workflows/release.yml

```diff
@@ -0,0 +1,46 @@
+name: release
+
+on:
+  push:
+    branches:
+      - master
+      - 'prerelease-**'
```

Comment:
Would it be possible to add extra branch protection on branches starting with `prerelease-*` and restrict push access to Form3 members only?

Reply:
I used prerelease only for testing, will remove that entry